test/python/test_rticonnextdds_connector.py

```diff
@@ -20,6 +20,13 @@ class TestConnector:
     :class:`rticonnextdds_connector.Connector` object.
     """
 
+    # This must be the first test in this file.
+    def test_setting_max_objects_per_thread(self):
```

Comment:
Can't we just delete this test? The other PR is removing this option because it's no longer required to set the max objects per thread.